$NetBSD$

Makefiles use LIBGNOMEUI_CFLAGS etc, so make sure that the values are
added to the right variable.

--- configure.orig	2009-12-21 19:47:53.000000000 +0000
+++ configure
@@ -24931,8 +24931,8 @@ pkg_failed=no
 echo $ECHO_N "checking for GNOMEUI... $ECHO_C" >&6; }
 
 if test -n "$PKG_CONFIG"; then
-    if test -n "$GNOMEUI_CFLAGS"; then
-        pkg_cv_GNOMEUI_CFLAGS="$GNOMEUI_CFLAGS"
+    if test -n "$LIBGNOMEUI_CFLAGS"; then
+        pkg_cv_LIBGNOMEUI_CFLAGS="$LIBGNOMEUI_CFLAGS"
     else
         if test -n "$PKG_CONFIG" && \
     { (echo "$as_me:$LINENO: \$PKG_CONFIG --exists --print-errors \"libgnomeui-2.0 >= \$LIBGNOMEUI_REQUIRED\"") >&5
@@ -24940,7 +24940,7 @@ if test -n "$PKG_CONFIG"; then
   ac_status=$?
   echo "$as_me:$LINENO: \$? = $ac_status" >&5
   (exit $ac_status); }; then
-  pkg_cv_GNOMEUI_CFLAGS=`$PKG_CONFIG --cflags "libgnomeui-2.0 >= $LIBGNOMEUI_REQUIRED" 2>/dev/null`
+  pkg_cv_LIBGNOMEUI_CFLAGS=`$PKG_CONFIG --cflags "libgnomeui-2.0 >= $LIBGNOMEUI_REQUIRED" 2>/dev/null`
 else
   pkg_failed=yes
 fi
@@ -24949,8 +24949,8 @@ else
 	pkg_failed=untried
 fi
 if test -n "$PKG_CONFIG"; then
-    if test -n "$GNOMEUI_LIBS"; then
-        pkg_cv_GNOMEUI_LIBS="$GNOMEUI_LIBS"
+    if test -n "$LIBGNOMEUI_LIBS"; then
+        pkg_cv_LIBGNOMEUI_LIBS="$LIBGNOMEUI_LIBS"
     else
         if test -n "$PKG_CONFIG" && \
     { (echo "$as_me:$LINENO: \$PKG_CONFIG --exists --print-errors \"libgnomeui-2.0 >= \$LIBGNOMEUI_REQUIRED\"") >&5
@@ -24958,7 +24958,7 @@ if test -n "$PKG_CONFIG"; then
   ac_status=$?
   echo "$as_me:$LINENO: \$? = $ac_status" >&5
   (exit $ac_status); }; then
-  pkg_cv_GNOMEUI_LIBS=`$PKG_CONFIG --libs "libgnomeui-2.0 >= $LIBGNOMEUI_REQUIRED" 2>/dev/null`
+  pkg_cv_LIBGNOMEUI_LIBS=`$PKG_CONFIG --libs "libgnomeui-2.0 >= $LIBGNOMEUI_REQUIRED" 2>/dev/null`
 else
   pkg_failed=yes
 fi
@@ -25030,8 +25030,8 @@ To get pkg-config, see <http://pkg-confi
 See \`config.log' for more details." >&2;}
    { (exit 1); exit 1; }; }
 else
-	GNOMEUI_CFLAGS=$pkg_cv_GNOMEUI_CFLAGS
-	GNOMEUI_LIBS=$pkg_cv_GNOMEUI_LIBS
+	LIBGNOMEUI_CFLAGS=$pkg_cv_LIBGNOMEUI_CFLAGS
+	LIBGNOMEUI_LIBS=$pkg_cv_LIBGNOMEUI_LIBS
         { echo "$as_me:$LINENO: result: yes" >&5
 echo "${ECHO_T}yes" >&6; }
 	:
